includes/class-gp-openai-review.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+class GP_OpenAI_Review {
+	/**
+	 * The OpenAI key.
+	 *
+	 * @var string
+	 */
+	const OPENAI_KEY = '';
```

Comment:
This should not be a const but set through a filter so that we can set it on translate.wordpress.org.